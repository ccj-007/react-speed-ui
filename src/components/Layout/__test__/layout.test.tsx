import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Layout } from '../../index';

describe('test Layout component', () => {
  it('should render', () => {
    const { asFragment } = render(<Layout />);
    expect(asFragment()).toMatchSnapshot();
  });
});
