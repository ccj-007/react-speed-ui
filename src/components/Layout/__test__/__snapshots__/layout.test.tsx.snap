// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Layout component should render 1`] = `
<DocumentFragment>
  <section
    class="speed-layout"
  />
</DocumentFragment>
`;
